### PR TITLE
Add enabled setting to README

### PR DESCRIPTION
**Pull request by luabud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,10 @@ There are several settings you can configure to customize the behavior of this e
             <td>flake8.ignorePatterns</td>
             <td><code>[]</code></td>
             <td>Configure [glob patterns](https://docs.python.org/3/library/fnmatch.html) as supported by the fnmatch Python library to exclude files or folders from being linted with Flake8.</td>
+        <tr>
+            <td>flake8.enabled</td>
+            <td><code>true</code></td>
+            <td>Specifies whether to enable or disable linting Python files using Flake8. This setting can be applied globally or at the workspace level. If disabled, the linting server itself will continue to be active and monitor read and write events, but it won't perform linting or expose code actions. </td>
```
Comment (cwebster-99) [inline README.md:87]: I think Code Actions is supposed to be capitalized based on other VS Code docs [(ref)](https://code.visualstudio.com/docs/editor/refactoring#_code-actions-quick-fixes-and-refactorings)
```suggestion
            <td>Specifies whether to enable or disable linting Python files using Flake8. This setting can be applied globally or at the workspace level. If disabled, the linting server itself will continue to be active and monitor read and write events, but it won't perform linting or expose Code Actions. </td>
```

Comment (luabud) [reply README.md]: great catch, thank you!!

File: package.nls.json [hunk 2]
```diff
@@ -16,4 +16,4 @@
     "settings.showNotifications.onError.description": "Show notifications for errors.",
     "settings.showNotifications.onWarning.description": "Show notifications for errors and warnings.",
     "settings.showNotifications.always.description": "Show all notifications."
-}
+}
```
Comment (karthiknadig) [inline package.nls.json:19]:
```suggestion
}

```

**Review by karthiknadig — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -21,9 +21,9 @@ The Flake8 extension provides features to improve your productivity while workin
 
 ### Disabling Flake8
 
-You can skip linting with Flake8 for specific files or directories by setting the `flake8.ignorePatterns` setting.
+You can skip linting with Flake8 for specific files or directories by setting the `flake8.ignorePatterns` setting.
```
Comment (brettcannon) [inline README.md:24]:
```suggestion
You can skip linting with Flake8 for specific files or directories by setting the `flake8.ignorePatterns` setting.
```

**Review by brettcannon — APPROVED** [review]

**Pull request merged**